From 3d98e3f91c4b29be6540a5c51814ac053d84a1d5 Mon Sep 17 00:00:00 2001
From: Xinhuang Li <buluess.li@rock-chips.com>
Date: Mon, 22 Oct 2018 14:27:37 +0800
Subject: [PATCH] arm64: dts: rockchip: rename codec for rk3328 android

fix the vts error like this:
Failed to parse! Parsing error at token LexToken(COMMA,',',1,48) in line 1;
explain:
the vts will parsing the codec'name after android P, the comma is not allow
in the codec'name

Change-Id: I6cb011331368d64417bd9955ba5d3deb84d49b8d
Signed-off-by: Xinhuang Li <buluess.li@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dtsi   | 6 +++---
 arch/arm64/boot/dts/rockchip/rk3328-evb-android.dtsi    | 6 +++---
 arch/arm64/boot/dts/rockchip/rk3328-rock64-android.dtsi | 4 ++--
 3 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dtsi b/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dtsi
index a6280c3fb281..876f253544b7 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328-box-liantong.dtsi
@@ -34,7 +34,7 @@
 		compatible = "simple-audio-card";
 		simple-audio-card,format = "i2s";
 		simple-audio-card,mclk-fs = <256>;
-		simple-audio-card,name = "rockchip,rk3328";
+		simple-audio-card,name = "rockchip-rk3328";
 		simple-audio-card,cpu {
 			sound-dai = <&i2s1>;
 		};
@@ -47,7 +47,7 @@
 		compatible = "simple-audio-card";
 		simple-audio-card,format = "i2s";
 		simple-audio-card,mclk-fs = <128>;
-		simple-audio-card,name = "rockchip,hdmi";
+		simple-audio-card,name = "rockchip-hdmi";
 		simple-audio-card,cpu {
 			sound-dai = <&i2s0>;
 		};
@@ -85,7 +85,7 @@
 
 	spdif-sound {
 		compatible = "simple-audio-card";
-		simple-audio-card,name = "rockchip,spdif";
+		simple-audio-card,name = "rockchip-spdif";
 		simple-audio-card,cpu {
 			sound-dai = <&spdif>;
 		};
diff --git a/arch/arm64/boot/dts/rockchip/rk3328-evb-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3328-evb-android.dtsi
index f5524b633d3a..c5b520197c25 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-evb-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328-evb-android.dtsi
@@ -34,7 +34,7 @@
 		compatible = "simple-audio-card";
 		simple-audio-card,format = "i2s";
 		simple-audio-card,mclk-fs = <256>;
-		simple-audio-card,name = "rockchip,rk3328";
+		simple-audio-card,name = "rockchip-rk3328";
 		simple-audio-card,cpu {
 			sound-dai = <&i2s1>;
 		};
@@ -47,7 +47,7 @@
 		compatible = "simple-audio-card";
 		simple-audio-card,format = "i2s";
 		simple-audio-card,mclk-fs = <128>;
-		simple-audio-card,name = "rockchip,hdmi";
+		simple-audio-card,name = "rockchip-hdmi";
 		simple-audio-card,cpu {
 			sound-dai = <&i2s0>;
 		};
@@ -58,7 +58,7 @@
 
 	spdif-sound {
 		compatible = "simple-audio-card";
-		simple-audio-card,name = "rockchip,spdif";
+		simple-audio-card,name = "rockchip-spdif";
 		simple-audio-card,cpu {
 			sound-dai = <&spdif>;
 		};
diff --git a/arch/arm64/boot/dts/rockchip/rk3328-rock64-android.dtsi b/arch/arm64/boot/dts/rockchip/rk3328-rock64-android.dtsi
index 7345381fe419..cd7c7adf8a70 100644
--- a/arch/arm64/boot/dts/rockchip/rk3328-rock64-android.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3328-rock64-android.dtsi
@@ -62,7 +62,7 @@
 		compatible = "simple-audio-card";
 		simple-audio-card,format = "i2s";
 		simple-audio-card,mclk-fs = <256>;
-		simple-audio-card,name = "rockchip,rk3328";
+		simple-audio-card,name = "rockchip-rk3328";
 		simple-audio-card,cpu {
 			sound-dai = <&i2s1>;
 		};
@@ -75,7 +75,7 @@
 		compatible = "simple-audio-card";
 		simple-audio-card,format = "i2s";
 		simple-audio-card,mclk-fs = <128>;
-		simple-audio-card,name = "rockchip,hdmi";
+		simple-audio-card,name = "rockchip-hdmi";
 		simple-audio-card,cpu {
 			sound-dai = <&i2s0>;
 		};
-- 
2.35.3

